variables-cluster.tf

```diff
@@ -29,6 +29,12 @@ variable "control_plane_is_public" {
   type        = bool
 }
 
+variable "assign_public_ip_to_control_plane" {
+  default     = false
+  description = "An additional variable to toggle public IP on Kubernetes control plane endpoint when its public."
```

Comment:
Change description to: 

"Whether to assign a public IP address to the API endpoint for public access. Requires the control plane subnet to be public to assign a public IP address."

Reply:
@hyder -Fixed